### PR TITLE
`UpdateThrottlerConfig --unthrottle-app ...`

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

Followup to https://github.com/vitessio/vitess/pull/13351, we introduce `UpdateThrottlerConfig --unthrottle-app ...`; example:

```sh
$ vtctldclient UpdateThrottlerConfig --unthrottle-app="vreplication" commerce
```

`--unthrottle-app` immediately removes any throttling rule from the app. It's a short form for the following syntax:

```sh
$ vtctldclient UpdateThrottlerConfig --throttle-app="vreplication" --throttle-app-duration=0 commerce
```

`endtoend` tests now using this new flag.

## Related Issue(s)

- https://github.com/vitessio/vitess/issues/13341
- docs: https://github.com/vitessio/website/pull/1539

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on the CI
-   [ ] Documentation was added or is not required

### CODE REVIEW

**Comment by shlomi-noach** [comment]
Docs PR: https://github.com/vitessio/website/pull/1539

File: go/vt/vtctl/vtctl.go [hunk 1]
```diff
@@ -702,7 +702,7 @@ var commands = []commandGroup{
 			{
 				name:   "UpdateThrottlerConfig",
 				method: commandUpdateThrottlerConfig,
-				params: "[--enable|--disable] [--threshold=<float64>] [--custom-query=<query>] [--check-as-check-self|--check-as-check-shard] [--throttle-app=<name>] [--throttle-app-ratio=<float, range [0..1]>] [--throttle-app-duration=<duration>] <keyspace>",
+				params: "[--enable|--disable] [--threshold=<float64>] [--custom-query=<query>] [--check-as-check-self|--check-as-check-shard] [--throttle-app|unthrottle-app=<name>] [--throttle-app-ratio=<float, range [0..1]>] [--throttle-app-duration=<duration>] <keyspace>",
```
Comment (mattlord) [inline go/vt/vtctl/vtctl.go:705]: Any reason why we describe the two flags differently in vtctlclient than vtctldclient?

**Review by mattlord — COMMENTED** [review]
Can we please add/update a unit or e2e test for unthrottling? Preferably unit but either is fine.

**Comment by shlomi-noach** [comment]
> Can we please add/update a unit or e2e test for unthrottling? Preferably unit but either is fine.

The PR includes a e2e test change, please take another look 🙏

File: go/vt/vtctl/vtctl.go (hunk 1 above)
Comment (shlomi-noach) [reply go/vt/vtctl/vtctl.go:705]: Just an oversight. I was missing one `push`. Fixed.

**Comment by mattlord** [comment]
> > Can we please add/update a unit or e2e test for unthrottling? Preferably unit but either is fine.
> 
> The PR includes a e2e test change, please take another look 🙏

Ah, I see. The logical `unthrottle` e2e test code now actually uses the new flag. 👍

**Review by mattlord — APPROVED** [review]

**Comment by shlomi-noach** [comment]
Consistent failures in `onlineddl_vrepl` `TestSchemaChange/throttled_and_unthrottled_migration` which is good news as it shows the tests are relevant.

**Review by GuptaManan100 — APPROVED** [review]

**Pull request merged**